### DISCUSSION TITLE
Flash devices Rf-Star CC2652P

### DISCUSSION THREAD

**Discussion by Max-zigbee2mqtt** [Koenkk/zigbee2mqtt#11219] [Devices]
Hello,
I successfully flashed the zigbee Rf-Star module
RF-BM-2652P2 
CC2652P with the latest firmware available on the Koenkk site 
Z-Stack-firmware, can you help me identify the TX and RX ports on the motherboard.
I thank you for your help.

**Comment by TomasWeij** [comment]
Hi Max,

How did you flash this module? I can't seem to get it working with openocd or smartrf04eb
